inc/namespace.php

```diff
@@ -216,3 +220,20 @@ function load_plugins() {
 		require dirname( __DIR__ ) . '/plugins/' . $file;
 	}
 }
+
+/**
+ * Disable the install plugins / themes capability in Cloud, as it's not possible to do in that context.
```

Comment:
This is too long, split to short description + long desc.

Reply:
```suggestion
 * Disable the install plugins / themes capability.
 * 
 * In the cloud context it's not possible to do this so this hides 
 * non functional UI.
```